pkg/apimachinery/discoverysummarizer/types.go

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Moved to `cmd/kubernetes-discovery/discoverysummarizer/types.go`
This is not the api exposed by discovery summarizer server.